source/_integrations/osoenergy.markdown

```diff
@@ -23,10 +23,26 @@ This OSO Energy integration uses a subscription key, which a user can create for
 
 ## Platforms
 
+### Sensor
+
+The `osoenergy` sensor integration exposes OSO Energy data as a sensor.
+
+The platform exposes the following sensors:
+
+- Heater Mode for water heaters.
+- Optimization Mode for water heaters.
+- Power load (kW) for water heaters.
+- Tapping capacity (kWh) for water heaters.
+- Capacity mixed water at 40°C (L) for water heaters.
+- V40 Min (L) for water heaters.
```

Comment:
what is V40 Min (L)? What does this do?

Reply:
V40 Min (L) is the minimum allowed quantity of mixed hotwater at 40 degrees C for the heater. 
It serves as a guard to turn on the heating if the temperature of the water drops below that treshold - when the capacity drops below the v40 min.
This guard can change depending on the operations on the water heater. For example it can drop to 0 is a user activates sleep mode.

Reply:
Thank you for the explanation.
So the first few words before "for water heater" is the actual name of the sensor? Could you maybe add a brief explanation for each sensor, like you just did? I think this would improve the value of the documentation. At least for me, the names alone don't say it all.

Reply:
I have added descriptions to the sensors.

Reply:
Nice :star_struck: Thank you!